decidim-core/lib/decidim/events/base_event.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require_relative "../../../app/helpers/decidim/sanitize_helper"
```

Comment:
This doesn't seem to be necessary in `develop`. Do you know why it may be happening? Is that some change in how the autoloader is working between versions? 

I know it may be out of the scope of the current PR, but what would be the "correct"/"elegant" way of solving this? Moving Decidim::SanitizeHelper from `decidim-core/app/helpers/decidim/sanitize_helper.rb` to `decidim-core/lib/decidim/sanitize_helper.rb`?

Reply:
This is not needed in develop, because a while back we implemented #10682 which took care of the issue. I am against moving the `sanitize_helper` as it would create a difference between 0.27 and 0.28. 
The non elegant solution seems to be the least effort that we need to do in order to backport this PR.

Reply:
Maybe I need more 🧉 because I don't grasp how the refactor on `decidm-dev`/`DummyResource` applies to `BaseEvent`. Are you sure you're referencing the correct PR?

Reply:
Back in #10682 we have moved the class definition of `Decidim::DummyResourceEvent` from `decidim-dev/lib/decidim/dev/test/rspec_support/component.rb` to its own class. This translated into the fact that the Decidim::DummyResourceEvent will be enabled and called after the full Decidim environment has been loaded. 

In 0.28 we rely on Zeitwerk to load DummyResourceEvent where we need it, as a result BaseEvent will also be called when needed, resolving the dependencies. 
In 0.27 we have the class definition in  `decidim-dev/lib/decidim/dev/test/rspec_support/component.rb` , which is loaded by `decidim-dev/lib/decidim/dev/test/spec_helper.rb` line 16, yet in this point of the application we do not have the autoloader available.

Reply:
Great explanation, thanks for the hand holding!!